README.md

```diff
@@ -1,3 +1,51 @@
+# MCAN
+
+The M\_CAN is a CAN IP module that can be realized as a standalone
+device, as part of an ASIC or on an FPGA. It performs communication
+according to ISO11898-1:2015. It supports Classical CAN and CAN FD
+(CAN with Flexible Data-rate). Additional transceiver hardware is
+required for connection to the CAN physical layer. The message
+storage is intended to be a single or dual-ported Message RAM
+outside of the module. It is connected to the M\_CAN via the Generic
+Master Interface. Depending on the chosen integration, multiple M\_CAN
+controllers may share the same Message RAM. The Host CPU is connected
+via the 32-bit Generic Interface.[^1]
+
+[^1]: [Bosch M\_CAN](https://www.bosch-semiconductors.com/ip-modules/can-ip-modules/m-can/)
```

Comment:
If the text is quoted from the product page, maybe it should be a block quote?

```suggestion
> The M\_CAN is a CAN IP module that can be realized as a standalone
> device, as part of an ASIC or on an FPGA. It performs communication
> according to ISO11898-1:2015. It supports Classical CAN and CAN FD
> (CAN with Flexible Data-rate). Additional transceiver hardware is
> required for connection to the CAN physical layer. The message
> storage is intended to be a single or dual-ported Message RAM
> outside of the module. It is connected to the M\_CAN via the Generic
> Master Interface. Depending on the chosen integration, multiple M\_CAN
> controllers may share the same Message RAM. The Host CPU is connected
> via the 32-bit Generic Interface.[^1]

[^1]: [Bosch M\_CAN](https://www.bosch-semiconductors.com/ip-modules/can-ip-modules/m-can/)
```

Reply:
Done